web/static/js/components/remote_retro.jsx

```diff
@@ -34,6 +34,7 @@ class RemoteRetro extends Component {
 
     return (
       <Room
+        currentUser={currentPresence}
```

Comment:
the misalignment in naming here makes me think we'll want to go with `currentPresence` as the prop name. that way, , up above, we won't introducee the weirdness of calling `currentUser.user`